### ISSUE TITLE
Error in Anaconda: undefined symbol:  PyFPE_jbuf

### ISSUE THREAD

**Issue by francescomalandrino** [MSeal/agglom_cluster#17]
I am running agglom_cluster in Anaconda Python 2.7 on linux. It install fine via Anaconda's "pip", but then I cannot import "hac":

----> 1 import hac                                                                                  
                                                                                                    
/home/francesco/anaconda2/lib/python2.7/site-packages/hac/__init__.py in <module>()                 
      1 __all__ = ['cluster', 'renamemap']                                                          
      2                                                                                             
----> 3 from .cluster import GreedyAgglomerativeClusterer                                           
                                                                                                    
ImportError: /home/francesco/anaconda2/lib/python2.7/site-packages/hac/cluster.so: undefined symbol:
 PyFPE_jbuf

I already tried removing and re-installing the whole anaconda, so I am fairly sure it's not a problem with messed-up library versions :)

**Comment by MSeal** [comment]
Sorry for the delayed fix, but 2.0.7 should work for you now. The issue was with an older version of Cython building a `.c` file which wasn't compiling cleanly. Works locally with anaconda on my linux box for py2 and py3.

```
08:46:33 mseal ~/Workspace (venv:.py3local)  
$ ~/anaconda2/bin/pip install --no-cache agglomcluster
Collecting agglomcluster
  Downloading https://files.pythonhosted.org/packages/40/5a/6675d8094915a2f22b53d2e57f2f58f753c7fa91ad813e05e4209a02dee5/AgglomCluster-2.0.7.tar.gz (326kB)
    100% |████████████████████████████████| 327kB 28kB/s 
Requirement already satisfied: networkx>1.7 in /home/mseal/anaconda2/lib/python2.7/site-packages (from agglomcluster)
Requirement already satisfied: decorator>=4.1.0 in /home/mseal/anaconda2/lib/python2.7/site-packages (from networkx>1.7->agglomcluster)
Installing collected packages: agglomcluster
  Running setup.py install for agglomcluster ... done
Successfully installed agglomcluster-2.0.7
You are using pip version 9.0.1, however version 10.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
08:47:11 mseal ~/Workspace (venv:.py3local)  
$ ~/anaconda2/bin/python
Python 2.7.14 |Anaconda, Inc.| (default, Dec  7 2017, 17:05:42) 
[GCC 7.2.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import hac
>>>
```

**Comment by francescomalandrino** [comment]
I can confirm it works :) Thanks a lot!
